pybind11.BUILD

```diff
@@ -34,3 +34,9 @@ cc_library(
     includes = ["include"],
     deps = ["@local_config_python//:python_headers"],
 )
+
+config_setting(
+    name = "darwin",
+    values = {"cpu": "darwin"},
```

Comment:
I'm not sure `darwin` is a valid `cpu` value, and even if it is, the recommended approach is to use platforms. For example:
```
config_setting(
    name = "macos",
    constraint_values = ["@platforms//os:osx"],
)
```